qCC/ccClippingBoxTool.cpp

```diff
@@ -428,6 +428,12 @@ ccHObject* GetSlice(ccHObject* obj, ccClipBox* clipBox, bool silent)
 			return nullptr;
 		}
 		clipBox->flagPointsInside(inputCloud, &selectionTable);
+
+		if (invert)
```

Comment:
Could you maybe add a 'invert' boolean parameter to the 'flagPointsInside' method and do this directly there?

(or maybe even rename the method ```flagPoints(inputCloud, &selectionTable, bool inside);```?

I can do it if you prefer.

Reply:
Makes more sens, I'll do it